client/editor/map_editor.cpp

```diff
@@ -184,9 +187,21 @@ void map_editor::select_tool_tile()
     ett_wdg_active = false;
   } else {
     ui.tbut_tool_tile->setToolTip(_("Close Tile Tool"));
-    ui.stackedWidget_tools->addWidget(ett_wdg);
+    ett_wdg = new editor_tool_tile(0);
```

Comment:
How comes we create the `editor_tool_tile` from two different places? Can we avoid it? Where do we delete it?

Reply:
I had the creation here at first and then got a coverity error so I moved it to the constructor. I added while troubleshooting the stacked widget. Let me see if I can remove it and also ensure I clear/free it when we close the dialog.

Reply:
Setting it to nullptr in the constructor should make coverity happy I guess.

Reply:
got rid of the extra "new" call